src/server/modules/pageinfo.ts

```diff
@@ -0,0 +1,322 @@
+import { defineNuxtModule } from '@nuxt/kit'
+import fs from 'fs'
+import os from 'os'
```

Comment:
好みの問題かもですけど…

> npmからインストールしたサードパーティ製のモジュールとの区別が明確になるため、付けておくことが推奨されます。

https://jsprimer.net/use-case/nodecli/read-file/#fs-module

```suggestion
import fs from 'node:fs'
import os from 'node:os'
```

Reply:
修正します。可能なら再発防止策として eslint-plugin-unicorn とかを入れても良いかなあって思います。いつも忘れるので…

https://github.com/sindresorhus/eslint-plugin-unicorn
https://zenn.dev/sosukesuzuki/articles/ee9c612446c62d